crates/store/re_entity_db/src/entity_db.rs

```diff
@@ -140,10 +140,21 @@ impl EntityDb {
         self.store_info().map(|ri| &ri.application_id)
     }
 
-    pub fn recording_property<C: re_types_core::Component>(&self) -> Option<C> {
+    pub fn recording_property<C: re_types_core::Component>(
```

Comment:
There are only two properties that we query, maybe it makes sense to add direct accessors using their two descriptors?

Reply:
I'd love to but I believe you tried yourself in the past: `re_entity_db` doesn't know `re_types`